meson.build

```diff
@@ -1,17 +1,34 @@
 project('qfreerdp', 'cpp',
   version : '1.0.0',
-  meson_version : '>= 1.0',
+  meson_version : '>= 0.63',  # before version 0.63.0 Meson would fail to find Qt 6.1 or later
```

Comment:
I had troubles with the version shipped in Ubuntu 22.04 with `qt_deps.get_variable('QMAKE_SPEC')` it's 0.61 AFAICT, do you think `0.63` behaves better ? 
TBH that made me remove the distro meson and update it to the most recent version on all my test platforms and I've set `1.0` as what seemed like a safe value...

Reply:
We had the same issues, in fact, all of the testing was done on meson 1.3.2 installed from pip. But I didn't want to restrict your users if, according to meson's docs, 0.63 is enough: https://mesonbuild.com/Qt6-module.html

Reply:
Ok let's take that and we'll see if people complains